Source/Core/Quaternion.js

```diff
@@ -179,6 +181,7 @@ define([
      * Computes a rotation from the given heading, pitch and roll angles. Heading is the rotation about the
      * negative z axis. Pitch is the rotation about the negative y axis. Roll is the rotation about
      * the positive x axis.
+     * @deprecated since V1.38
```

Comment:
Does the "since V1.38" show up in the doc?  We usually just put `@deprecated`.

Reply:
I tested and it shows up in the doc. I found this syntax [here](http://usejsdoc.org/tags-deprecated.html)